strawberryfields/apps/plot.py

```diff
@@ -130,7 +130,10 @@ def graph(g: nx.Graph, s: Optional[list] = None, plot_size: Tuple = (500, 500)):
         import plotly.graph_objects as go
     except ImportError:
         raise ImportError(plotly_error)
-
+    
+    if (get_ipython().__class__.__name__ != 'ZMQInteractiveShell'):
+        plotly.io.renderers.default = "browser" 
```

Comment:
I believe `get_ipython` is only available in IPython terminals and notebooks, so this would raise an `NameError` if used in regular Python. It would need to be wrapped in a try-except block.

E.g., something like the following could work:
```python
try:
    in_notebook = get_ipython().__class__.__name__ == 'ZMQInteractiveShell'
except NameError:
    in_notebook = False

if not in_notebook:
    pio.renderers.default = "browser"
```
You also need to `import plotly.io as pio` in the above try-except block to be able to access the `pio.renderers.default` property. 🙂